acquia_lift.module

```diff
@@ -39,3 +38,38 @@ function acquia_lift_page_attachments(array &$page) {
   // Attach lift CSS to all pages.
   $page['#attached']['library'][] = 'acquia_lift/acquia-lift';
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function acquia_lift_toolbar() {
+  $items = [];
+
+  $items['acquia_lift'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'html_tag',
+      '#tag' => 'button',
+      '#value' => t('Acquia Lift'),
+      '#attributes' => [
+        'title' => t('Acquia Lift'),
+        'class' => ['toolbar-icon', 'toolbar-icon-acquia-lift'],
+        // This particular id will make sure the JS behavior is attached to the
+        // link.
+        'id' => ['openLiftLink'],
+        'aria-pressed' => 'false',
+      ],
+    ],
+    '#access' => AccessResult::allowedIfHasPermission(\Drupal::currentUser(), 'access acquia lift links'),
```

Comment:
I think we also need to ```&& $pathContext->shouldAttach()```. This is because shouldAttach() conditionally stops Experience Builder from loading on pages, then on those pages you'd also not wanting to see the "Acquia Lift" shortcut. In this way, we'd never reach ```alert('Sorry, Acquia Lift could not be found');```.